tests/unit/core/statistics/test_constraints.py

```diff
@@ -1563,11 +1563,21 @@ def test_dataset_constraints_serialization():
     set1 = set(["col1", "col2"])
     columns_match_constraint = columnsMatchSetConstraint(set1)
 
+    val_set = {(1, 2), (3, 5)}
```

Comment:
This function is far too long. Not necessarily a problem to fix in this PR, but we should at least file a ticket to refactor this.

Reply:
I've changed it now, you can take a look when you can.